bobweb/bob/image_generating_service.py

```diff
@@ -31,11 +30,11 @@ async def generate_using_openai_api(prompt: str, image_size: int = 1024) -> Imag
     openai_api_utils.ensure_openai_api_key_set()
 
     payload = {
-        "model": "dall-e-3",
+        "model": "gpt-image-1",
         "prompt": prompt,
-        "n": 1,  # for Dall-e 3 only 1 image is allowed per request
+        "background": "opaque",  # transparent is also possible now (change JPEG -> PNG)
```

Comment:
I think parameter `moderation=low` should be added to the request to make the filtering less restrictive

Reply:
Done.